src/styles.scss

```diff
@@ -0,0 +1,19 @@
+@import './prompts/styles/variables';
+@import './prompts/styles/anims';
+@import './prompts/styles/tooltip';
+@import './prompts/styles/toast';
+@import './prompts/styles/prompts';
+@import './prompts/styles/buttons';
+@import './prompts/styles/checkbox';
+@import './prompts/styles/banner';
+@import './prompts/styles/banner.compact';
+
+@import './prompts/bell/bell.scss';
+@import './prompts/dialog/dialog.scss';
+
+@import './prompts/background-mask.scss';
+@import './prompts/overlay.scss';
+
+body {
+    padding: 1em 11px 12px 13px;
```

Comment:
Not sure we should be including global body padding in the SDK, is this just for dev/debug?

Reply:
Ahh yeah just for dev there 👍